### PR TITLE
Added option to select target class to mutate

### PR DESCRIPTION
**Pull request by JKutscha**
Issue #164.

### CODE REVIEW

File: bundles/org.pitest.pitclipse.launch.ui/src/org/pitest/pitclipse/launch/ui/PitArgumentsTab.java [hunk 1]
```diff
@@ -38,37 +54,79 @@
...
+    private Button targetClassCheckBoxButton;
+    /**
+     * Text which holds the information about which classes should be mutated
+     */
+    private Text targetClassText;
+    /**
+     * Pattern which matches a String which contains packages separated by dots and
+     * end with a class.
+     */
+    private static final String CLASS_PATTERN = "\\w+(\\.\\w+)*\\.\\w+";
```
Comment (LorenzoBettini) [inline bundles/org.pitest.pitclipse.launch.ui/src/org/pitest/pitclipse/launch/ui/PitArgumentsTab.java:99]: Please keep in mind that `$` is a valid character in a class name, while they cannot start with a number...
maybe JDT itself already provides some validation under that respect?

Comment (LorenzoBettini) [reply bundles/org.pitest.pitclipse.launch.ui/src/org/pitest/pitclipse/launch/ui/PitArgumentsTab.java:99]: Actually it's handled by Java itself, please have a look at https://stackoverflow.com/questions/5205339/regular-expression-matching-fully-qualified-class-names 

`Pattern.compile("\\p{javaJavaIdentifierStart}\\p{javaJavaIdentifierPart}*\\.\\p{javaJavaIdentifierStart}\\p{javaJavaIdentifierPart}*)*");`

Comment (JKutscha) [reply bundles/org.pitest.pitclipse.launch.ui/src/org/pitest/pitclipse/launch/ui/PitArgumentsTab.java]: Looks good, thank you.

**Comment by JKutscha** [comment]
This is currently blocked by #152, because I implemented in this PR helper methods which allow running specific launch configuration. This would be needed to test this feature.

**Comment by LorenzoBettini** [comment]
@JKutscha why have you closed this?

**Comment by JKutscha** [comment]
> 
> 
> @JKutscha why have you closed this?

I need to work on that right now, because of my deadline and so I closed this to avoid complications like the ones happend on #152.

**Pull request closed without merging**